src/components/App/components/UserInput/UserInput.js

```diff
@@ -71,7 +71,6 @@ function UserInput(props) {
                   id: 'font-size',
                   type: 'number',
                   min: MIN_FONT_SIZE_IN_PX,
-                  max: MAX_FONT_SIZE_IN_PX,
```

Comment:
Can you please explain what behaviour you're trying to achieve. I'm confused what the intent is if we're bounding the input, but not the Preview